### PR TITLE
Updated italian translation

### PR DESCRIPTION
**Pull request by TopoDiFogna**
I've updated the translation and added missing ones

### CODE REVIEW

File: po/it/system-monitor.po [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Traduzione italiana per il pacchetto system-monitor.
 # Copyright (C) 2012 THE system-monitor'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the system-monitor package.
-# Federico Piazza fede.piazza92@gmail.com, 2012.
```
Comment (p91paul) [inline po/it/system-monitor.po]: Differently from the Last-Translator, I would not remove this attribution. Just add yours!

File: po/it/system-monitor.po [hunk 2]
```diff
@@ -207,7 +207,7 @@ msgid "other"
 msgstr "altro"
 
 msgid "nice"
-msgstr "nice"
```
Comment (p91paul) [inline po/it/system-monitor.po]: AFAIK, "nice" is a technical term and isn't translated in Italian. I'm Italian, but I never heard of "particolare" as a translation for "nice"; can you point to a reference for that?

Comment (TopoDiFogna) [inline po/it/system-monitor.po]: Nice is used as detailed in this case i think, particolare or particolareggiato should do the job

File: po/it/system-monitor.po (hunk 1 above)
Comment (TopoDiFogna) [inline po/it/system-monitor.po]: Added one more commit to restore previous translator as requested

File: po/it/system-monitor.po (hunk 2 above)
Comment (p91paul) [inline po/it/system-monitor.po]: Check [this](http://askubuntu.com/questions/399357/what-does-the-nice-value-mean-in-cpu-utilization-statistics): nice is the time spent on niced (i.e. high priority) tasks. I believe it should not be translated!

**Pull request merged**